### ISSUE TITLE
harvesting fees of either asset independently

### ISSUE THREAD

**Issue by aliatiia** [tonkers-kuma/strategy-rebalancer#1]
shouldn't [L152](https://github.com/tonkers-kuma/strategy-rebalancer/blob/ad859fe5a4c9fc92dbb6ce405ac9a1793ffc03e1/contracts/Rebalancer.sol#L152) of Rebalancer.sol be:

`if (_pooledA >= _debtA || _pooledB >= _debtB)`

because theoretically it's possible to to have accrued fees in one asset and not the other

**Comment by tonkers-kuma** [comment]
yes, it's true that fees are accrued by individual assets, but `if (_pooledA >= _debtA || _pooledB >= _debtB)` would incorrectly collect fees if the the pool was skewed by a trade.
i.e `debtA = 100, debtB = 100`
`pooledA = 110.1, pooledB = 90.1`
you can't determine how much of `110.1-100 = 10.1` but actually a gain due to fees

easiest way would be wait until pool equalizes back up for the opportunity to determine exact gains
i.e `debtA = 100, debtB = 100`
`pooledA = 100.1, pooledB = 100.1` -> `gain .1A, .1B`
